bots/twitterbot.go

```diff
@@ -197,10 +202,54 @@ func (bot *TwitterBot) HandleRegistration(username string, msg []byte) string {
 	return string(res)
 }
 
-// SendDM sends a Twitter direct message msg to the given Twitter screenname.
+// sendDM sends a Twitter direct message msg to the given Twitter screenname.
 // The sender screenname should be set to the bot's reserved Twitter handle.
-func (bot *TwitterBot) SendDM(screenname, msg string) error {
+func (bot *TwitterBot) sendDM(screenname, msg string) (*twitter.DirectMessage, error) {
 	params := &twitter.DirectMessageNewParams{ScreenName: screenname, Text: msg}
-	_, _, err := bot.client.DirectMessages.New(params)
-	return err
+	dm, _, err := bot.client.DirectMessages.New(params)
+	return dm, err
+}
+
+// deleteOldDMs deletes all prior DMs before the bot runs.
+func (bot *TwitterBot) deleteOldDMs() {
+	log.Println("[registration bot] Deleting old DMs ...")
+	params := &twitter.DirectMessageGetParams{Count: 200} // up to a maximum of 200.
```

Comment:
200 is the upper bound limited by Twitter API. 
`deleteRequestDMs` deletes the message using its ID (which is returned by the API). But the dev can terminate the bot before it deletes all the messages (There is a timer before calling `deleteRequestDMs`). That's why I think we need `deleteOldDMs` to clean up remaining messages.